docs/formats.md

```diff
@@ -251,12 +520,12 @@ Creates a JSON file of just the assets defined in the style dictionary.
 
 * * *
 
-### sketch/palette
+### sketch/palette 
```

Comment:
I have tried to use it, but for me didn't work. in the future we should check it, and if it's not correct just remove it

Reply:
@dbanksdesign I have never used this. Does it work? If not, we should submit a PR to fix or remove.

Reply:
I think it doesn't work. I'll test it properly and if it doesn't work I'll submit a PR to remove it.